tests/unit/util.ts

```diff
@@ -24,233 +24,278 @@ import { Handle } from '@dojo/interfaces/core';
 import * as util from '../../src/util';
 
 const TIMEOUT = 3000;
+let timerHandle: any = null;
 
 registerSuite('utility functions', {
-	createTimer: (function () {
-		let timer: Handle | null;
+	afterEach() {
+		timerHandle && clearTimeout(timerHandle);
+		timerHandle = null;
+	},
 
-		return {
-			afterEach() {
-				timer && timer.destroy();
-				timer = null;
+	tests: {
+		createTimer: (function () {
+			let timer: Handle | null;
+
+			return {
+				afterEach() {
```

Comment:
Is this (and the ones below) needed with the suite-level `afterEach()`?

Reply:
I fixed it.